### ISSUE TITLE
Interpolation for timeseries plotting

### ISSUE THREAD

**Issue by xRSquared** [pyrochlore/obsidian-tracker#11]
# Enhancement
It would be great if generated graphs would interpolate data.

For example, say we have the following data:

Day 1: 150
Day 3: 160

Then, the graph would interpolate the data and plot a point at day 2 with a value of 155.

**Comment by pyrochlore** [comment]
It wouldn’t be very hard to implement this kind of feature. But may I ask your usage?
I usually want to see the original data when I do tracking.

**Comment by xRSquared** [comment]
I use it to track my investments. It is quite time-consuming to enter data daily. I think this is helpful in more general cases like tracking weight.

**Comment by pyrochlore** [comment]
If you just want lines connected, try to set fillGap to true. Then the line graph will not show gaps from the missing data. Is this what you need?

**Comment by pyrochlore** [comment]
It is quite time-consuming to enter data daily.

About this, any idea the plug-in can do better for you?

**Comment by xRSquared** [comment]
> If you just want lines connected, you may set fillGap to true. Then the line graph will not show gaps from the missing data. Is this what you need?
> Though no interpolated points.

Yes this is what I was looking for. I some how missed this parameter in the documentation. However, I just tried to use the parameter and it doesn't seem to be working. Did I write the YAML incorrectly? 

> 
> ```tracker
> searchType: tag	
> searchTarget: investments
> 
> folder: /Repo/
> startDate:
> endDate:
> line:
>     title: "Investments"
>     xAxisLabel: Date
>     yAxisLabel: ""
>     fillGap: true		
> ```

**Comment by xRSquared** [comment]
> > It is quite time-consuming to enter data daily.
> 
> About this, any idea the plug-in can do better for you?

Once I can get the lines to connect, it will no longer be time-consuming!

**Comment by pyrochlore** [comment]
The YAML looks good. I will do some test when I am back home.

**Comment by pyrochlore** [comment]
It's a bug. I have fixed it and will release it in the next few days.

**Comment by pyrochlore** [comment]
The fix was just released with the new version 1.3.0.
If you still have problems with it, don't hesitate to ask it here.
